azurerm/internal/services/loganalytics/migration/workspace_v1_to_v2.go

```diff
@@ -0,0 +1,31 @@
+package migration
+
+import (
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/loganalytics/parse"
+)
+
+func WorkspaceV1ToV2() schema.StateUpgrader {
+	// V1 to V2 is the same as v1 to v0 - to workaround a historical issue where `resource_group` was
```

Comment:
super minor typo in comment
```suggestion
	// V1 to V2 is the same as v0 to v1 - to workaround a historical issue where `resource_group` was
```